spec.go

```diff
@@ -6,12 +6,14 @@ import (
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/giantswarm/k8sclient/k8scrdclient"
 )
 
 type Interface interface {
 	CRDClient() k8scrdclient.Interface
+	CtrlClient() client.Client
```

Comment:
It's not 3 letters :(

Reply:
I feel you. But `RESTClient` was already breaking it. Anyway, it looks like upstream tries to work with all runtime objects using a single client in the future. When this is true all other clients are not relevant anymore anyway. Let's see how this will play out.

Reply:
Yeah I hope we can use this single one in the future.